lib/nfs_common.pm

```diff
@@ -288,7 +288,7 @@ sub check_y2_nfs_func {
         systemctl('is-active nfs-server');
         systemctl('restart rpcbind');
         systemctl('is-active rpcbind');
-        systemctl('restart rpcbind.socket');
+        systemctl('restart rpcbind.socket', timeout =>120);
```

Comment:
Hi Huajian,
LGTM, but can I know the issue happen on zkvm or zvm?

Reply:
> Hi Huajian, LGTM, but can I know the issue happen on zkvm or zvm?
s390-kvm-sle12

Reply:
Then no problem.